src/databases/Mili/avtMiliFileFormat.C

```diff
@@ -622,7 +626,7 @@ avtMiliFileFormat::GetMesh(int ts, int dom, const char *mesh)
         else
         {
             fpts = new float[amt];
-            read_results(dbid[dom], ts+1, subrec, 1, &nodpos_str, vsize, amt, fpts);
+            read_results(dbid[dom], ts+1, subrec, 1, const_cast<char**>(&nodpos_str), vsize, amt, fpts);
```

Comment:
This API for the formal argument for `nodpos_str` seems funky...Is this ours or some TPL I/O lib? We're passing the address of a string which I assume is the name of the variable to be read. But, the API seems to want a char** too suggesting it might manipulate/alloc the string pointed to by the pointer past in and there is no `const` qualification apparently. If the callee can wind up allocating, we'll leak. Ok, I see this is Mili and we're overhauling that enrirely. I wouldn't worry about this then.

Reply:
I agree - if it was doing some sort of alloc, then it was bad behavior to begin with.